scripts/genEvent.ts

```diff
@@ -53,7 +53,7 @@ const RECORD = (id: string): SQSRecord => ({
 
 const EVENT = (): SQSEvent => {
   const rs: SQSRecord[] = []
-  for (let i = 0; i < 10; i++) {
```

Comment:
So this will only be requeuing 2 events at a time rather than the 10 it appeared to be doing before? Can you expand a bit on the motivation behind that?

Reply:
This is a test file, not used in prod. When you start it locally via `npm start`, this loop controls how many records are in the event. I had it set to 10 when I was trying to find a memory leak, but 2 is more manageable.